openreview/venue_request/process/matchingProcess.py

```diff
@@ -59,26 +59,25 @@ def process(client, note, invitation):
     else:
         no_profiles_status = matching_status.get('no_profiles')
         if no_profiles_status:
-            profiles_status=f'''
+            without_profiles_status = f'''
```

Comment:
I would prefer that without_profiles_status and no publications_status are arrays of emails or profile ids and move the text to the comment section.

Reply:
@celestemartinez  please let us know what you think.

Reply:
right now, the already_invited field looks like this:

`No recruitment invitation was sent to the following users because they have already been invited:
{'testvenue.org/Test_Venue/2022/Conference/Reviewers/Invited': ['celeste@mail.com']}`

Do you mean @melisabok you only want to have `['celeste@mail.com']` here? and the text in the comment?

Reply:
Yes. What do you think?

We can keep a string but having an array could be easier to parse the data.

Reply:
this change hasn't been made yet

Reply:
Done.